src/book.js

```diff
@@ -0,0 +1,12 @@
+export default function book(){
+    return `
+        <h1>Book</h1>
+            <ol>
+                <li><img src = "" alt = ""></li>
```

Comment:
I doubt that you want this image to be a part of an ordered list, right? That wouldn't make much sense.